src/rubyfmt.rb

```diff
@@ -584,7 +598,7 @@ def format_params(ps, params, open_delim, close_delim)
     ps.emit_ident(open_delim)
   end
 
-  bad_params = params[6..-1].any? { |x| !x.nil? }
+  bad_params = params[7..-1].any? { |x| !x.nil? }
```

Comment:
I am not sure if this line makes sense anymore? Can there be more than 8 elements in `params`?